manim/utils/cfgwriter.py

```diff
@@ -0,0 +1,81 @@
+from os import path
+import configparser
+
+from .config_utils import successfully_read_files
+
+from rich.console import Console
+from rich.progress import track
+from rich.color import Color
+
+
+def check_valid_colour(color):
+    """Checks whether the entered color is a valid color according to rich
+    Parameters
+    ----------
+    color : :class:`str`
+        The color to check whether it is valid.
+    Returns
+    -------
+    Boolean
+        Returns whether it is valid color or not.
+    """
+    try:
+        Color.parse(color) is None
+        return True
+    except:
+        return False
+
+
+def main():
+    console = Console()
+    config = configparser.ConfigParser()
+    config.read(
+        successfully_read_files
+    )
+    default = {
+        "logging.keyword": "bold yellow",
+        "logging.level.notset": "dim",
+        "logging.level.debug": "green",
+        "logging.level.info": "blue",
+        "logging.level.warning": "red",
+        "logging.level.error": "red bold",
+        "logging.level.critical": "red bold reverse",
+        "log.level": "",
+        "log.time": "cyan dim",
+        "log.message": "",
+        "log.path": "dim",
+    }
+    console.print(
+        "[yellow bold]Manim Logger Configuration Editor[/yellow bold]", justify="center"
+    )
+    console.print(
+        "[red]The default colour is shown as input Statement.\nIf left empty default value will be assigned.[/red]"
+    )
+    console.print(
+        "[magenta]Please follow the link for available styles.[/magenta][link=https://rich.readthedocs.io/en/latest/style.html]docs[/link]"
+    )
+    for key in default:
+        console.print("Enter the Style for %s" % key + ":", style=key, end="")
```

Comment:
Does `rich` suggest the use of the `%` operator? If I remember correctly, `logger` asks you to pass the string as well as the interpolators as arguments, and not use the `%` operator.

Reply:
Well this is a normal print statement. So `%` operator is allowed.